regolith/helpers/u_milestonehelper.py

```diff
@@ -123,12 +124,15 @@ def db_updater(self):
         if rc.index > 1:
             doc = all_milestones[rc.index-2]
             identifier = doc['identifier']
+            if rc.type:
+                doc.update({'type': ALLOWED_TYPES[rc.type]})
```

Comment:
it might be good to give a more elegant exit if rc.type is not one of the allowed type keys.  Wrap in a try-except clause and raise a ValueError, giving the user a helpful error message if it fails.

Reply:
ok! can you please check if I did it correctly?